components/utilities/ymodem/ry_sy.c

```diff
@@ -0,0 +1,241 @@
+/**
+  * Copyright (c) 2019 Fuzhou Rockchip Electronics Co., Ltd
+  *
+  * SPDX-License-Identifier: Apache-2.0
+  ******************************************************************************
```

Comment:
Please use the standard style of RT-Thread header files to write copy right information. The copyright is OK, but others, please keep the style. Thanks.